docs/deprecations.rst

```diff
@@ -0,0 +1,80 @@
+============
+Deprecations
+============
+
+This page documents
+the major deprecations
+in ScreenPy Selenium's life,
+and how to adjust your tests
+to keep them up to date.
+
+4.1.0 Deprecations
+==================
+
+Boolean Positional Arguments Deprecation
+----------------------------------------
+
+The following class constructors 
+have been marked deprecated 
+when using a positional boolean argument in the constructor. 
+Starting with version 5.0.0
+you will be required to provide keywords 
+for the following boolean arguments.
+
```

Comment:
Should we put a note here that this isn't how these Actions were envisioned to be used, but just in case someone is using them this way...?

Reply:
Should that maybe go in the deprecation documentation?